app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] Nice working on your `create method` for creating a comment, Kindly check when I tried to create a comment it just loads without adding the comment.